contracts/script/SetUpEigenDA.s.sol

```diff
@@ -25,7 +25,7 @@ import "forge-std/StdJson.sol";
 // forge script script/Deployer.s.sol:SetupEigenDA --rpc-url $RPC_URL  --private-key $PRIVATE_KEY --broadcast -vvvv
 contract SetupEigenDA is EigenDADeployer, EigenLayerUtils {
 
-    string deployConfigPath = "script/eigenda_deploy_config.json";
+    string deployConfigPath = "script/input/eigenda_deploy_config.json";
```

Comment:
had to change this path to make the kurtosis devnet work (otherwise uploading the config would overwrite the rest of the directory.

I think I changed everywhere else that depended on this but please double check. Also I feel like we might want to have this path as an arg to the run function instead of hardcoded?